nutanix/data_source_nutanix_floating_ips_list.go

```diff
@@ -0,0 +1,242 @@
+package nutanix
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	v3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+	"github.com/terraform-providers/terraform-provider-nutanix/utils"
+)
+
+func dataSourceNutanixFloatingIPsList() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceNutanixFloatingIPsListRead,
+		Schema: map[string]*schema.Schema{
+			"length": {
```

Comment:
Do we need offset and length as part of the schema ? 
I guess in other datasources we iteratively fetch all the entities, combine them and populate the .tf file. 

Keeping offset and length in terraform file doesn't make sense to me. If you have any specific reasons for doing it we can discuss.